test/src/helpers/util.test.js

```diff
@@ -11,3 +13,38 @@ describe('util', () => {
     });
   });
 });
+
+describe('crypto-hash', () => {
+  it('Should return same hash', async () => {
+    const hash = await cryptoModule.encrypt('mypassword');
+    assert.deepStrictEqual(hash, await cryptoModule.encrypt('mypassword'));
+  });
+});
+
+describe('crypto-hash', () => {
+  it('Should return true', async () => {
+    const hash = await cryptoModule.encrypt('mypassword');
+    assert.equal(true, await cryptoModule.compare('mypassword', hash));
+  });
+});
+
+describe('crypto-hash', () => {
+  it('Should return false', async () => {
+    const hash = await cryptoModule.encrypt('mypassword');
+    assert.equal(false, await cryptoModule.compare('myPassword', hash));
+  });
+});
+
+describe('crypto-hash', () => {
+  it('Should return false', async () => {
+    const hash = await bcryptModule.encrypt('mypassword');
+    assert.equal(false, await bcryptModule.compare('myPassword', hash));
+  });
+});
+
+describe('crypto-hash', () => {
+  it('Should return false', async () => {
+    const hash = await bcryptModule.encrypt('mypassword');
+    assert.equal(true, await bcryptModule.compare('mypassword', hash));
```

Comment:
Usually the first parameter is what we are testing and the second one the expected. Not vice versa